easybuild/easyconfigs/g/gpustat/gpustat-0.6.0-gcccuda-2020b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonBundle'
+
+name = 'gpustat'
+version = '0.6.0'
+
+homepage = 'https://github.com/wookayin/gpustat'
+description = 'dstat-like utilization monitor for NVIDIA GPUs'
+
+toolchain = {'name': 'gcccuda', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('psutil', '5.8.0', {
+        'checksums': ['0c9ccb99ab76025f2f0bbecf341d4656e9c1351db8cc8a03ccd62e318ab4b5c6'],
+    }),
```

Comment:
We should be able to remove this. We have psutil 5.7.2 in the Python install and this version of gpustat was released long before psutil 5.8.0 was released.